src/main/java/service/ReportCreator.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ReportCreator {
+    List<String> create(Map<String, Integer> fruitTotalBalanceMap);
```

Comment:
```suggestion
    List<String> create();
```
We can have access to Storage directly, this method argument is redundant.